options.go

```diff
@@ -25,9 +27,13 @@ type Options struct {
 	Filesystem fs.FS
 
 	// Commonly modified options.
-	Verbose  bool
+	Verbose bool
+
 	LockMode LockMode
 
+	// CustomLocker overrides the default locker.
+	CustomLocker Locker
```

Comment:
IMO it could be named a bit better (just `Locker` may be a bit missleading - I wanted to emphasize that it's not required)